### PR TITLE
Add more scenerios for viostor/vioscsi test

### PR DESCRIPTION
**Pull request by suqinhuang**
Add scenerios for virstor/vioscsi test
e.g run iozone in system disk/data disk before/during/after sub tests (shutdown, reboot)

### CODE REVIEW

**Comment by suqinhuang** [comment]
ID: 1231022 1231021 1231021 1231005 1230684 1230683

**Comment by suqinhuang** [comment]
# cat job.log | grep -r -i "PASS smp_1.4096m" | wc -l
48

all 48 cases pass

**Comment by xutian** [comment]
can you add description about your PR?

**Comment by CongSmile** [comment]
From the cfg name viostor_in_use.cfg, it should be virtio_block only, but actually it has covered virtio_scsi.

File: qemu/tests/cfg/viostor_in_use.cfg [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+            format_disk = yes
+            images += " stg"
+            image_name_stg = "images/storage"
+            image_size_stg = 4G
+            force_create_image_stg = yes
+            remove_image_stg = yes
+    variants:
+        - with_block:
+            driver_name = viostor
+            drive_format = virtio
```
Comment (CongSmile) [inline qemu/tests/cfg/viostor_in_use.cfg]: Where does param "drive_format" be used?

Comment (suqinhuang) [inline qemu/tests/cfg/viostor_in_use.cfg]: +    type = driver_in_use 

 81     driver = params["driver_name"]

**Comment by CongLi** [comment]
Missing param "target_process".

**Comment by suqinhuang** [comment]
no need target process

**Comment by CongLi** [comment]
@suqinhuang Why no need of "target_process" ? which is used to check the backgroud test status in guest.

File: qemu/tests/cfg/viostor_in_use.cfg [hunk 2]
```diff
@@ -0,0 +1,62 @@
+- viostor_in_use:
+    type = driver_in_use
+    start_vm = yes
+    kill_vm_on_error = yes
+    check_guest_bsod = yes
+    login_timeout = 240
+    suppress_exception = True
+    driver_load_cmd = C:\devcon.exe enable @DRIVER_ID
+    driver_unload_cmd = C:\devcon.exe disable @DRIVER_ID
+    driver_id_cmd = C:\devcon.exe find * | find "VirtIO"
```
Comment (CongLi) [inline qemu/tests/cfg/viostor_in_use.cfg]: Where are the 3 lines above used?

File: qemu/tests/cfg/viostor_in_use.cfg [hunk 3]
```diff
@@ -0,0 +1,62 @@
+- viostor_in_use:
```
Comment (CongLi) [inline qemu/tests/cfg/viostor_in_use.cfg]: From the cfg name viostor_in_use.cfg, it should be virtio_block only, but actually it has covered virtio_scsi.
I think need update the case name.

Others look good to me.

Comment (suqinhuang) [inline qemu/tests/cfg/viostor_in_use.cfg]: how about vio_storage_in_use ?

File: qemu/tests/cfg/virtio_storage_in_use.cfg [hunk 4]
```diff
@@ -0,0 +1,62 @@
+- virtio_storage_in_use:
+    type = driver_in_use
+    start_vm = yes
+    kill_vm_on_error = yes
+    check_guest_bsod = yes
+    login_timeout = 240
+    suppress_exception = True
+    driver_load_cmd = C:\devcon.exe enable @DRIVER_ID
+    driver_unload_cmd = C:\devcon.exe disable @DRIVER_ID
+    driver_id_cmd = C:\devcon.exe find * | find "VirtIO"
```
Comment (CongLi) [inline qemu/tests/cfg/virtio_storage_in_use.cfg]: Where are the 3 lines above used?

File: qemu/tests/cfg/virtio_storage_in_use.cfg [hunk 5]
```diff
@@ -0,0 +1,62 @@
...
+    only Windows
+    variants:
+        - before_bg_test:
+            run_bg_flag = "before_bg_test"
+        - during_bg_test:
+            run_bg_flag = "during_bg_test"
+        - after_bg_test:
+            run_bg_flag = "after_bg_test"
+    variants:
+        - with_stop_continue:
```
Comment (CongLi) [inline qemu/tests/cfg/virtio_storage_in_use.cfg]: suppress_exception should be False for this scenario.

File: qemu/tests/cfg/virtio_storage_in_use.cfg [hunk 6]
```diff
@@ -0,0 +1,64 @@
+- virtio_storage_in_use:
+    type = driver_in_use
+    start_vm = yes
+    kill_vm_on_error = yes
+    check_guest_bsod = yes
+    login_timeout = 240
+    suppress_exception = no
+    driver_load_cmd = C:\devcon.exe enable @DRIVER_ID
+    driver_unload_cmd = C:\devcon.exe disable @DRIVER_ID
+    driver_id_cmd = C:\devcon.exe find * | find "VirtIO"
```
Comment (CongLi) [inline qemu/tests/cfg/virtio_storage_in_use.cfg]: Where are the 3 lines above used?

**Comment by suqinhuang** [comment]
@CongLi Thanks, could  you review again

**Comment by CongLi** [comment]
Acked-by: Cong Li <coli@redhat.com>

**Comment by pingl2016** [comment]
#/home/staf-kvm-devel/workspace/avocado-vt/scripts/scan_results.py | grep PASS | wc -l
50
After test the patch with windows guest, 48 cases passed, other 2 cases are guest install and image check.
Acked-by: Ping Li pingl@redhat.com

**Comment by CongLi** [comment]
Thanks @suqinhuang @pingl2016 .

**Pull request merged**